docs/en/security-privacy-considerations.rst

```diff
@@ -0,0 +1,499 @@
+ Credential within the Wallet is stored securely; it verifies the overall security of the Wallet environment, within which the Credentials reside. Therefore, while the attestation supports the Verifier's 
+ confidence that the Credential comes from a secure source, it is ultimately a broad assurance of the Wallet's security, rather than a specific validation of individual Credential storage. 
+
+SR-V-20
+~~~~~~~
+.. list-table:: 
+   :widths: 8 92
+
+   * - |check-icon|
+     - (conditional w.r.t.  I-10) The Verifier must ensure that the Credential was issued by an Issuer that only issues Credentials to trustworthy Wallets.
+
+By checking the trust of the Issuer, the Verifier ensures that the Credential was issued by a trusted Issuer committed to issuing Credentials only to secure Wallets (as for SR-I-50).
+
+.. note::
+  Currently, no explicit security and privacy measures related to this requirement are specified in [`OpenID4VC-SecTrust`_], it remains a TODO. 
+
+SR-W-20
+~~~~~~~
+.. list-table:: 
+   :widths: 8 92
+
+   * - |check-icon|
+     - The Wallet must provide trustworthy and complete information about Issuers to the User.
+
+The Wallet Instance discovers the trusted Issuers using trusted third party resources, such as the Federation API (e.g., using the Subordinate Listing Endpoint of the Trust Anchor and its Intermediates), 
+inspecting the Issuer metadata and Trust Marks for filtering the PID Provider.
+
+The Issuer's information is displayed to the User during the issuance process and can be subsequently read by the User as it is inside the issued Credential. 
+In addition to the Issuer's information, the Digital Credential Metadata Type also contains information on the Authentic Source.
+
+SR-W-30
+~~~~~~~
+.. list-table:: 
+   :widths: 8 92
+
+   * - |check-icon|
+     - The Wallet must provide trustworthy and complete information about Verifiers to the User.
+
+The Wallet validates the Trust Chain related to the Verifier and its information is displayed to the User before the presentation.
+
+Privacy Requirements
+--------------------
+
+PR-CF-30
+~~~~~~~~
+.. list-table:: 
+   :widths: 8 92
+
+   * - |check-icon|
+     - The Credential Format must ensure that there is a robust mechanism to ensure that data that is not to be released to a Verifier cannot be extracted by the Verifier (selective disclosure).
+
+Both SD-JWT and mDOC-CBOR provide selective disclosure capability, allowing Holders to reveal only specific fields to the Verifier. 
+
+PR-CF-40
+~~~~~~~~
+.. list-table:: 
+   :widths: 8 92
+
+   * - |partially-check-icon|
+     - The Credential Format must support correlation protection.
+
+While selective disclosure is a strong tool for preventing correlation, full unlinkability is not guaranteed in all cases. Issues like Verifier collusion or Issuer tracking can arise, especially if sensitive identifiers (e.g., taxpayer ID) are disclosed. 
```

Comment:
according to our knowledge, even an undisclosed taxpayer id value used in the same issuer signed credential allows tracking.

Reply:
this is not an issue only related to taxpayer id value, but to the entire credential

considerting to produce an hash of the issuer signed credential, the traceability can be achieved as well.

Reply:
ok, so we can probably delete the end of the sentence. Is this ok?

> , especially if sensitive identifiers (e.g., taxpayer ID) are disclosed.